src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/RoleAssignmentTests.cs

```diff
@@ -78,7 +78,14 @@ public void RaByResource()
         {
             ResourcesController.NewInstance.RunPsTest("Test-RaByResource");
         }
-        
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void RaValidateInputParameters()
```

Comment:
unit test added, but basically it is doing same thing as the powershell scripts:)

---
In reply to: [118623381](https://github.com/Azure/azure-powershell/pull/4019#discussion_r118623381) [](ancestors = 118623381)